### PR TITLE
[13.0][MIG] purchase_order_product_recommendation : Migration to 13.0

### PR DESCRIPTION
**Pull request by manuelcalerosolis**
This is the migration of purchase_order_product_recommendation to 13.0

@Tecnativa TT21652

### CODE REVIEW

File: purchase_order_product_recommendation/README.rst [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+* Tecnativa
+
+Contributors
+~~~~~~~~~~~~
+
+* `Tecnativa <https://www.tecnativa.com>`_:
+
+  * David Vidal
+  * Ernesto Tejeda
```
Comment (sergio-teruel) [inline purchase_order_product_recommendation/README.rst:91]: @manuelcalerosolis You add yourself to contributor's file but I think that after you do not regenerate this file.

File: purchase_order_product_recommendation/tests/test_recommendation.py [hunk 2]
```diff
@@ -0,0 +1,306 @@
+# Copyright 2019 David Vidal <david.vidal@tecnativa.com>
+# Copyright 2020 Manuel Calero - Tecnativa
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from datetime import date, datetime
```
Comment (sergio-teruel) [inline purchase_order_product_recommendation/tests/test_recommendation.py]: Use fields.Date or fields.Datetime

File: purchase_order_product_recommendation/tests/test_recommendation.py [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+        )
+        cls.wh2 = cls.env["stock.warehouse"].create(
+            {"name": "TEST WH2", "code": "TST2"}
+        )
+        # Locations
+        location_obj = cls.env["stock.location"]
+        cls.supplier_loc = location_obj.create(
+            {"name": "Test supplier location", "usage": "supplier"}
+        )
+        cls.customer_loc = location_obj.create(
```
Comment (sergio-teruel) [inline purchase_order_product_recommendation/tests/test_recommendation.py:56]: Why do you need a customer location for this module?

File: purchase_order_product_recommendation/tests/test_recommendation.py [hunk 4]
```diff
@@ -0,0 +1,306 @@
...
+        # Create deliveries and receipts orders to have a history
+        cls.picking_obj = cls.env["stock.picking"]
+        cls.picking_1 = cls.picking_obj.create(
+            {
+                "location_id": cls.wh1.lot_stock_id.id,
+                "location_dest_id": cls.customer_loc.id,
+                "partner_id": cls.partner.id,
+                "picking_type_id": cls.wh1.out_type_id.id,
+            }
+        )
```
Comment (sergio-teruel) [inline purchase_order_product_recommendation/tests/test_recommendation.py:68]: Why do you need a out picking location for this module?

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

File: purchase_order_product_recommendation/README.rst (hunk 1 above)
Comment (manuelcalerosolis) [reply purchase_order_product_recommendation/README.rst:91]: Done!

File: purchase_order_product_recommendation/tests/test_recommendation.py (hunk 4 above)
Comment (manuelcalerosolis) [reply purchase_order_product_recommendation/tests/test_recommendation.py:68]: They were in version 12.0 I have simply respected it

File: purchase_order_product_recommendation/tests/test_recommendation.py (hunk 2 above)
Comment (manuelcalerosolis) [reply purchase_order_product_recommendation/tests/test_recommendation.py]: Changed

File: purchase_order_product_recommendation/tests/test_recommendation.py (hunk 4 above)
Comment (sergio-teruel) [reply purchase_order_product_recommendation/tests/test_recommendation.py:68]: Ok.

**Review by sergio-teruel — APPROVED** [review]

**Review by pedrobaeza — APPROVED** [review]
There's some changes that belong to black in the migration commit, but it's now very difficult to fix that, so let's continue

**Pull request merged**